pylint/pyreverse/utils.py

```diff
@@ -292,7 +293,34 @@ def check_graphviz_availability():
     if shutil.which("dot") is None:
         print(
             "The requested output format is currently not available.\n"
-            "Please install 'Graphviz' to have other output formats "
-            "than 'dot' or 'vcg'."
+            "Please install 'Graphviz' to have image output formats."
+        )
+        sys.exit(32)
+
+
+def check_if_graphviz_supports_format(output_format: str) -> None:
+    """Check if the ``dot`` command supports the requested output format.
+
+    This is needed if image output is desired and ``dot`` is used to convert
+    from *.gv into the final output format.
+    """
+    dot_output = subprocess.run(
+        ["dot", "-T?"], capture_output=True, check=False, encoding="utf-8"
+    )
+    match = re.match(
+        pattern=r".*Use one of: (?P<formats>(\S*\s?)+)",
+        string=dot_output.stderr.strip(),
+    )
+    if not match:
+        print(
+            "Unable to determine Graphviz supported output formats. "
+            "Pyreverse will continue, but subsequent error messages "
+            "regarding the output format may come from Graphviz directly."
+        )
+        return
```

Comment:
``mypy`` forced me to include this. 😉 
Relying on the textual output of the ``dot`` command to determine the supported output formats is probably not the best solution, but I'd argue the tradeoff for more user friendly error messages and ability to fail fast is worth it. 
But if you have concerns or feedback please feel free to address it.